template_feed/Microsoft.DotNet.Test.ProjectTemplates.8.0/content/NUnit-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Aleksei Kharlov aka halex2005 (codeofclimber.ru)",
+  "classifications": [ "Test", "NUnit" ],
+  "name": "NUnit 3 Test Project",
+  "defaultName": "NUnitTestProject",
```

Comment:
`defaultName` is mentioned twice in NUnit templates, only one entry should be kept

Reply:
Fixed.